### DISCUSSION TITLE
about installing opentopas

### DISCUSSION THREAD

**Discussion by jxaflxb** [OpenTOPAS/OpenTOPAS#49] [Validation]
[output.txt](https://github.com/user-attachments/files/16163885/output.txt)
### Contact Details

jxaflxb@sina.com

### Describe your simulation result

I decided to upgrade topas  from ver 3.0 to ver 4.0. Following the installation instruction，opentopas was compiled successfully with several warnings & without errors. but when running rundemos.csh in topas directory. it confused me with its  test results & hints which not present in topas ver 3.9，like the smooth graph、Dicom images，subsequently the topas-nbio will not be compiled,  i turned
 to geant4 forum ,but it failed to fix this. how can opentopas run demos.csh successfully. here is the terminal output, macOS Sonoma, Apple M2 Chip.[
[output.txt](https://github.com/user-attachments/files/16163885/output.txt)
](url)

### Version

4.0

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by JoseRamosMendez** [comment]
The link to the log output is broken, would you please share it again?

**Reply by jxaflxb** [reply]
the output text is updated again!

**Comment by JoseRamosMendez** [comment]
Thaks for sharing the logfile. Most of the tests in that script have not enabled Qt (b:Ts/UseQt = "True"), the script was done to work with only OpenGL. But as now, the live visualization can only be done through Qt (we will update such file).
  
It looks like (from your logfile) the examples/Nozzle/ScanningTargetMovingHorizontal.txt worked fine, isn't it? You can try other examples by just adding b:Ts/UseQt = "True" and run them one-by-one.

**Reply by jxaflxb** [reply]
That't it after checking per *.txt in demos, Appreciate for your careful  check.by adding “b:Ts/UseQt = "True" in *。txt，the visualization of demos.csh perform successfully in turn which performed in ver.3.9
